finetune/utils.py

```diff
@@ -233,6 +234,35 @@ def fit_transform(self, y):
         return self.transform(y)
 
 
+class RegressionEncoder:
+    def __init__(self):
+        self.classes_ = None
+
+    def fit(self, x):
+        self.fit_transform(x)
+        return self
+
+    def transform(self, x):
+        output = np.array(x)
+        rank = len(output.shape)
+        if rank == 1:
+            return np.expand_dims(output, 1)  # for single output value regression.
+        if rank == 2:
+            return output
+        raise ValueError("Unresolvable shape: {}. Must be able to fit a format [batch, n_outputs]".format(output.shape))
+
+    def fit_transform(self, x):
+        output = self.transform(x)
+        self.classes_ = np.arange(output.shape[1])
```

Comment:
Nitpicking but naming convention  should probably be something other than classes_.